back/app/policies/initiative_policy.rb

```diff
@@ -12,13 +12,10 @@ def initialize(user, scope)
     def resolve
       not_draft = scope.where(publication_status: %w[published closed])
 
-      if user&.admin?
+      if user && UserRoleService.new.can_moderate_initiatives?(user)
```

Comment:
I couldn't use `can_moderate?`, as suggested in this [Slack thread](https://citizenlabco.slack.com/archives/C05H2FX706M/p1689856565833729), as it requires a specific `Initiative` object in its parameters.